### ISSUE TITLE
what kinds of tools are welcomed here?

### ISSUE THREAD

**Issue by matkoniecz** [osmlab/osm-tagging#2]
> Eventually this repo may also contain code and tools for gaining better insights into tagging and making changes to tags.

( from https://lists.openstreetmap.org/pipermail/tagging/2018-June/037322.html )

what kinds of tools are welcomed here? I guess that preferable method for that would be opening issue proposing to include something followed by a PR.

What are minimal requirements? I would expect something like

- license
- documentation how one may use tool
- dependencies of tool specified in way standard for a given programming language
- working tool

For example I recently made tool for analyzing impact of unauthorized automated edits (example output at https://gist.github.com/matkoniecz/23bd62764cac4ed81984bb1f8af52dc8 )

I am currently planning to make a tool with statistics about sources of specific tag - as requested on https://lists.openstreetmap.org/pipermail/talk/2018-June/080808.html

I may also cleanup and share some code used to make edits listed at https://wiki.openstreetmap.org/wiki/Mechanical_Edits/Mateusz_Konieczny_-_bot_account

I like idea of sharing such code as it would allow to reuse code and maybe somebody would spot how my code may be improved.

**Comment by bhousel** [comment]
Good question - I admit that I hadn't thought very far along on this.. Maybe we should just add links to existing tools in the README for now?

**Comment by bhousel** [comment]
Yeah I thought about this more.. Because any tools we add are likely to be different languages and licenses, for now lets just link to other repos in the README instead of trying to collect code here.

It would be great to have your tools included @matkoniecz! 😄
